composer.json

```diff
@@ -10,13 +10,13 @@
         }
     ],
     "require": {
-        "php": ">=5.5.0",
+        "php": "^5.5|^7.0|^8.0",
         "ext-curl": "*",
-        "illuminate/support": "~5.0|~6.0|~7.0|~8.0"
+        "illuminate/support": "^5.0|^6.0|^7.0|^8.0"
```

Comment:
This all seems fine. The only thing I would say is that we should replace the single `|` with ` || `, as, having checked the Composer docs to confirm I understand all the syntax, I realised that `|` isn't documented as it is considered deprecated. Might as well use the "correct" OR operator.